.config/tsaoptions.json

```diff
@@ -0,0 +1,9 @@
+{
+    "instanceUrl": "https://msazure.visualstudio.com",
+    "projectName": "One",
+    "areaPath": "One\\MGMT\\Compute\\Powershell\\Powershell\\PowerShell Core",
```

Comment:
We should update this path but the new path needs to be created.  We can do this in another PR.